pluto/umbrel-app.yml

```diff
@@ -0,0 +1,23 @@
+manifestVersion: 1
+id: pluto
+name: Pluto
+category: bitcoin
+version: "0.6.0-rc.1"
+tagline: Pluto
```

Comment:
You can add a short tagline here so that users can see at a glance what Pluto is when they are browsing the app store. For example, you could do something like:

`Manage and monitor open-source bitcoin mining devices`